### DISCUSSION TITLE
The mimir log is garbled or the file is damaged？

### DISCUSSION THREAD

**Discussion by Alexis-cmyk-a11y** [grafana/mimir#4139] [Help and support] [answered]
After the log file is opened
![image](https://user-images.githubusercontent.com/82816977/216236303-df0fc97b-f3f8-477f-bc53-7b4684b05a9d.png)

**Comment by pstibrany** [comment, marked as answer]
This is expected. This isn't human-readable log, rather a file where Mimir stores ongoing activities when it runs. If some activity is found there after Mimir restarts, Mimir will log messages to its standard log, assuming that theses activities ran when Mimir crashed.

**Reply by Alexis-cmyk-a11y** [reply]
How to open Mimir's standard log

**Reply by pstibrany** [reply]
By Mimir's "standard log" I meant all log messages that Mimir writes to standard error output (file descriptor 2). It's up to the environment to redirect it to the correct place (ie. file).
